phpstan.neon.dist

```diff
@@ -1,9 +1,4 @@
 parameters:
   level: 'max'
-  ignoreErrors:
```

Comment:
removing reason?

Reply:
Because I found a way to overcome this error
```php
/** File AbstractTypedIDEntity.php */
final public function __construct
...
```